docs/platforms/python/integrations/celery/crons.mdx

```diff
@@ -9,6 +9,10 @@ Sentry Crons allows you to monitor the uptime and performance of any scheduled,
 
 Use the Celery integration to monitor your [Celery periodic tasks](https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html) and get notified when a task is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.
 
+<Note>
+Please note that monitors will only be created on the task's first run and will not be recurring.
```

Comment:
Our upsert will re-create monitors if they are deleted through the UI but kept in code. What do you think we remove this part?

```suggestion
Please note that monitors will only be created on the task's first run.
```